ecs-cli/modules/compose/cli/ecs/app/command.go

```diff
@@ -51,7 +51,7 @@ const (
 func ComposeCommand(factory ProjectFactory) cli.Command {
 	return cli.Command{
 		Name:   "compose",
-		Usage:  "Execute docker-compose style commands on an ECS cluster.",
+		Usage:  "Executes docker-compose–style commands on an ECS cluster.",
```

Comment:
The added hyphen there is an en-dash, not hyphen, which can look odd in some terminals.